src/src/ex7/client/src/redux/selectors/itemsEntitiesSelectors.js

```diff
@@ -0,0 +1,6 @@
+const getItemsEntities = (state) => state.itemsEntities;
+export const getItems = (state) => getItemsEntities(state).items;
+export const getSearchInput = (state) => state.itemsEntities.searchInput;
+export const getTasksStatusState = (state) => state.itemsEntities.tasksStatus;
+export const getDoneItems = (state) =>
```

Comment:
not sure if you ever explained about it, but selectors should be quick access without any calculations.
If you need to create a selector with complex calculations (such as here with the filter) you should use this library https://www.npmjs.com/package/reselect that will create a cached layer and will calculate the selector only once.
But you don't need to change it right now as its a bit advanced flow :)